### PR TITLE
add recipe for darts

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rluria14** [comment]
@ndmaxar @oblute agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/darts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name.replace("pt.", "") }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/khanrc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b0c4689cdabc5fdec0ba3c829057aa2cb94f9a97036cf5ae545de6247d301e9c
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (dopplershift) [inline recipes/darts/meta.yaml:14]: It looks like this is pure Python? You should be able to make this `noarch: python`, remove the `skip`, and we can ignore the build failure due to the lack of PyTorch on Windows.

File: recipes/darts/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/darts/LICENSE:1]: This file should already be in the tarball from GitHub? In that case, you shouldn't include the file in the recipe.

File: recipes/darts/meta.yaml (hunk 1 above)
Comment (rluria14) [reply recipes/darts/meta.yaml]: noarch added. skip removed.

File: recipes/darts/LICENSE (hunk 2 above)
Comment (rluria14) [reply recipes/darts/LICENSE:1]: weirdly it does not include the in the tarball.

Comment (dopplershift) [reply recipes/darts/LICENSE:1]: That is bizarre, but ok.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**